lib/index.js

```diff
@@ -45,18 +44,18 @@ function addRoutes(app) {
   app.post(cfg.routes.verify, brRest.when.prefers.ld,
     asyncHandler(async (req, res) => {
       const {challenge, domain, presentation} = req.body;
+      const suite = [new Ed25519Signature2018(), new Ed25519Signature2020()];
       let _report;
       try {
         _report = await vc.verify({
-          documentLoader: loader.documentLoader,
+          documentLoader: _documentLoader,
           challenge,
           domain,
-          suite: new Ed25519Signature2018(),
+          suite,
           presentation,
           checkStatus
         });
       } catch(e) {
-        console.log('VC.VERIFY ERROR', e);
```

Comment:
If the logging is removed (which is good), this try/catch becomes useless.

Reply:
Removed f36950d